src/main/kotlin/apply/application/ApplicantService.kt

```diff
@@ -56,6 +59,34 @@ class ApplicantService(
         }
     }
 
+    fun resetPassword(applicantPasswordFindInformation: ApplicantPasswordFindInformation): String {
+        return when (
+            applicantRepository.existsByNameAndEmailAndBirthday(
+                applicantPasswordFindInformation.name,
+                applicantPasswordFindInformation.email,
+                applicantPasswordFindInformation.birthday
+            )
+        ) {
+            true -> updatePassword(applicantPasswordFindInformation, randomStringGenerator.generateRandomString())
+            else -> throw ApplicantValidateException()
+        }
+    }
```

Comment:
if-else로 바꿔보았는데 가독성 측면에서 어떤게 더 좋을까용

뭔가 when의 근본이 switch임을 감안하면 여러 분기로 나눠질 때에 사용하는게 바람직할거 같다는 생각도 들구 말이죠

```suggestion
    fun resetPassword(applicantPasswordFindInformation: ApplicantPasswordFindInformation): String {
        return if (
            applicantRepository.existsByNameAndEmailAndBirthday(
                applicantPasswordFindInformation.name,
                applicantPasswordFindInformation.email,
                applicantPasswordFindInformation.birthday
            )
        ) updatePassword(applicantPasswordFindInformation, randomStringGenerator.generateRandomString()) 
        else throw ApplicantValidateException()
    }
```

Reply:
when이 남들볼때 편하게 보일것 같아서 해보았어요. 둘중에 팀원들 볼때 편한 코드로 적용하면 좋을거같아요